nest/02.introduction.md

```diff
@@ -0,0 +1,57 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all of the files coming with a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage the packages**.
+It is similar to an app store for any smartphone, or the Windows Store for example. One of the main differences between those and Nest (and most package managers), is that the upgrading must be manually commanded.
+On a smartphone, the applications are usually automatically upgraded overnight when you're charging your phone, if an update is available.
+Here, it's up to the user to upgrade their installed packages whenever they want or need to.
+
+To use Nest, the only interface available for now is the Command Line Interface (CLI).
+It might be a bit nerve racking for someone who has never seen a terminal or typed any command before, but don't worry, this guide is here to help you feel comfortable!
+
+## 1.2 Packages and dependencies
+Often, a package requires another one to work properly.
+Say, for example, someone develops a messenging application, which relies on encrypted messages to provide communication between two users.
+This application will probably use a package, written by someone else, to encrypt the messages sent with their app.
+We say that such messenging application **depends** on that encryption package, because without that package, the application would probably not work at all.
+Now, if you want that application to work properly, you'll also need to install the package they used to encrypt messages.
+
+If a package is required by another one to work properly, that package is called a **dependency**.
+Whenever you install a package via a package manager (and Nest as well), it will check if the package in question has any dependencies.
+And if it does, it will install only the dependencies that are not already installed on the OS.
+So, when you install a package, don't be surprised if you see other packages getting installed too, they are just dependencies needed for that package to work properly.
+
+## 1.3 Repositories
+Packages are stored in repositories.
+Think of a repository as a big folder containing the packages.
```

Comment:
"folder" is not a unix term. Also I'm not sure "repository" needs analogy given the original meaning of the word.